deploy.sh

```diff
@@ -14,6 +15,18 @@ gitCommitSubject=$(git log -1 --pretty='%h %s')
 
 declare -A loggedin
 
+if [[ "$1" == "REDEPLOY" ]]; then
```

Comment:
if [[ -z "$1" ]]; then

Reply:
Given `$1` would not be empty I don't think this works. $1 has the file names on partial deploys

Reply:
Or rather, if you want this to work you'd need to remove the `REDEPLOY` from the workflow file

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/1950/files#r980139225